app/src/androidTest/java/net/skyscanner/backpack/snackbar/BpkSnackbarTests.kt

```diff
@@ -41,14 +43,16 @@ class BpkSnackbarTests : BpkSnapshotTest() {
   }
 
   @Test
-  fun defaultWithAction() {
+  @Variants(BpkTestVariant.Default)
```

Comment:
that has layout + colour implications, so all 4?

Reply:
We test it all in titleWithIconAndAction